src/Microsoft.Azure.WebJobs/ExtensionAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Description
+{
+    /// <summary>
+    /// Attribute applied to the <see cref="IExtensionConfigProvider"/> implementation for an extension.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
```

Comment:
Should this be marked as `Inherited ` false? It would make sense tho prevent that from happening.